website/templates/emails/withdrawal_request_declined.html.mako

```diff
@@ -6,13 +6,28 @@
     <%!
         from website import settings
     %>
-        Dear ${requester.fullname},<br>
-        <br>
-        Your request to withdraw your ${document_type} <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. Login and visit your ${document_type} to view their feedback. The ${document_type} is still publicly available on ${reviewable.provider.name}.
+        % if document_type == 'preprint':
+            Dear ${requester.fullname},<br>
+            <br>
+            Your request to withdraw your preprint <a href="${reviewable.absolute_url}">"${reviewable.title}"</a> from ${reviewable.provider.name} has been declined by the service moderators. Login and visit your ${document_type} to view their feedback. The ${document_type} is still publicly available on ${reviewable.provider.name}.
```

Comment:
preprint -> `${document_type}` for the same reason that `document_type == 'preprint'` doesn't work. :(